### DISCUSSION TITLE
Any way to get the device's serial number by using api?

### DISCUSSION THREAD

**Discussion by someone132s** [vladimirs-git/fortigate-api#38] [Q&A]
as title

**Comment by DDinVA** [comment]
GET https://<fortigate_ip>/api/v2/cmdb/system/status

**Comment by someone132s** [comment]
Big thank! I discovered that the serial number is actually included in the top-level fields of every API response, outside of the results section.

________________________________
发件人: inetexpert ***@***.***>
发送时间: 2024年12月28日 3:59
收件人: vladimirs-git/fortigate-api ***@***.***>
抄送: someone132s ***@***.***>; Author ***@***.***>
主题: Re: [vladimirs-git/fortigate-api] Any way to get the device's serial number by using api? (Discussion #38)


GET https://<fortigate_ip>/api/v2/cmdb/system/status

―

**Comment by nomad-cyanide** [comment]
Otherwise use "api/v2/monitor/system/ha-statistics" to get the serial numbers of all cluster-nodes.
